src/editors/containers/ProblemEditor/data/MarkDownParser.js

```diff
@@ -38,64 +38,64 @@ function getHints(markdown) {
 }
 
 function getShortAnswerOptions(markdown) {
-// TODO: Include tolerence for number it fails for now.
-let problemType = ProblemTypeKeys.TEXTINPUT;
-  if ((!markdown || !markdown.trim())) {
-    return {
-      shortAnswersList: [],
-      problemType,
-    };
-  }
-  const markdownListData = markdown.split('\n');
-  const shortAnswersList = [];
-  let gotAnswer = false;
-  for (const i in markdownListData) {
-    const row = markdownListData[i].trim();
-    const feedbackStart = row.indexOf('{{');
-    const feedbackEnd = row.indexOf('}}');
-    let feedback = '';
-    let answer = '';
-    let correct = true;
-    if (row.startsWith('=') && !gotAnswer) {
-      gotAnswer = true;
-      if (feedbackStart > -1) {
-        feedback = row.slice(feedbackStart + 2, feedbackEnd);
-        answer = row.slice(1, feedbackStart).trim();
-      } else {
-        answer = row.slice(1).trim();
-      }
-    } else if (row.startsWith('=') && gotAnswer) {
-      break;
-    } else if (gotAnswer && row.startsWith('or=')) { // the next correct answer
-      if (feedbackStart > -1) {
-        answer = row.slice(3, feedbackStart).trim();
-        feedback = row.slice(feedbackStart + 2, feedbackEnd);
-      } else {
-        answer = row.slice(3).trim();
+  // TODO: Include tolerence for number it fails for now.
```

Comment:
`tolerence for number` = tolerance for numerical input problems?

Reply:
@connorhaugh  yes, don't have a provision for that  with markdown yet, also I have a skeleton structure of OLX parser getting ready.

Reply:
OLX parser is merged here